src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +82,26 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createExceptionRecord(
```

Comment:
is this method supposed to just transform the exception record to case format? The method name is confusing, creating an Exception record from ExceptionRecord?

Reply:
fair point. will think of a better name unless provided earlier. since this PR is preview of what's coming and therefore based from the one before - #573

Reply:
A better name maybe should be validateExceptionRecordTransformation

Reply:
this method will wrap the whole thing:

transform, create case, return new case id